src/components/TabMenu/Item/Item.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import styles from './Item.module.css';
+
+export interface ItemProps extends React.HTMLAttributes<HTMLLIElement> {
+  active?: boolean;
+}
+
+const Item: React.FC<ItemProps> = ({
+  className: customClassName,
+  active = false,
+  children,
+  onClick = () => {},
```

Comment:
@pixelbandito @sebastianvera @dkordik As I opened this PR, I noticed that this `onClick` handler is simply dropping any ability to listen to click events on this component. Seems like a bug. Thoughts on removing this?

Reply:
does seem odd that this prop is given a default value if it's not actually used in the component... seems safe to remove as it's unused. (I wonder if the blame would reveal anything interesting- maybe it was formerly used and never cleaned up?)